iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/IConsensus.java

```diff
@@ -207,4 +207,11 @@ public interface IConsensus {
    * @return consensusGroupId list
    */
   List<ConsensusGroupId> getAllConsensusGroupIds();
+
+  /**
+   * Return all consensus group ids from disk.
```

Comment:
Add some comments just like
We need to parse all the RegionGroupids from the disk directory before starting the consensus layer, so getAllConsensusGroupIds returns an empty list, so we need to add a new interface

Reply:
sure